crypto/rsa/rsa_pss.c

```diff
@@ -118,13 +126,15 @@ int RSA_verify_PKCS1_PSS_mgf1(RSA *rsa, const unsigned char *mHash,
                        "expected: %d retrieved: %d", sLen,
                        maskedDBLen - i);
         goto err;
+    } else {
+        sLen = maskedDBLen - i;
     }
     if (!EVP_DigestInit_ex(ctx, Hash, NULL)
         || !EVP_DigestUpdate(ctx, zeroes, sizeof(zeroes))
         || !EVP_DigestUpdate(ctx, mHash, hLen))
         goto err;
-    if (maskedDBLen - i) {
-        if (!EVP_DigestUpdate(ctx, DB + i, maskedDBLen - i))
+    if (sLen) {
```

Comment:
style nit: `sLen != 0`

Reply:
Updated.

Maybe you need to re-approve this PR?